blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -78,6 +68,12 @@ export default class Pipelines extends Component {
                                 })
                             }
                         </Table>
+                        
+                        { pipelines && pipelines.$pager &&
+                            <button disabled={!pipelines.$pager.hasMore} className="btn-show-more btn-secondary" onClick={() => pipelines.$pager.fetchMore()}>
+                                {pipelines.$pending ? 'Loading...' : 'Show More'}
```

Comment:
Maybe the button `disabled` attribute should also be influenced by `pipelines.$pending`? Seems like it should also be disabled when "Loading ..." ?